code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -95,10 +99,255 @@
 	new /obj/item/reagent_containers/food/pill/initropidril(src)
 	new /obj/item/flashlight/flare/glowstick/red(src)
 
+//////////////////
+/* Common Boxes */
+//////////////////
+/obj/item/storage/box/tapes
+	name = "Tape Box"
+	desc = "A box of spare recording tapes"
+	icon_state = "tape_box"
+
+/obj/item/storage/box/tapes/populate_contents()
+	for(var/I in 1 to 6)
+		new /obj/item/tape/random(src)
+
+/obj/item/storage/box/cups
+	name = "box of paper cups"
+	desc = "It has pictures of paper cups on the front."
+	icon_state = "cup_box"
+
+/obj/item/storage/box/cups/populate_contents()
+	for(var/I in 1 to 7)
+		new /obj/item/reagent_containers/food/drinks/sillycup(src)
+
+/obj/item/storage/box/drinkingglasses
+	name = "box of drinking glasses"
+	desc = "It has a picture of drinking glasses on it."
+
+/obj/item/storage/box/drinkingglasses/populate_contents()
+	for(var/I in 1 to 6)
+		new /obj/item/reagent_containers/food/drinks/drinkingglass(src)
+
+/obj/item/storage/box/condimentbottles
+	name = "box of condiment bottles"
+	desc = "It has a large ketchup smear on it."
+
+/obj/item/storage/box/condimentbottles/populate_contents()
+	for(var/I in 1 to 6)
+		new /obj/item/reagent_containers/food/condiment(src)
+
+/obj/item/storage/box/mousetraps
+	name = "box of Pest-B-Gon mousetraps"
+	desc = "<B><FONT color='red'>WARNING:</FONT></B> <I>Keep out of reach of children</I>."
+	icon_state = "mousetraps_box"
+
+/obj/item/storage/box/mousetraps/populate_contents()
+	for(var/I in 1 to 6)
+		new /obj/item/assembly/mousetrap(src)
+
+/obj/item/storage/box/ids
+	name = "spare IDs"
+	desc = "Has so many empty IDs."
+	icon_state = "id_box"
+
+/obj/item/storage/box/ids/populate_contents()
+	for(var/I in 1 to 7)
+		new /obj/item/card/id(src)
+
+/obj/item/storage/box/lights
+	name = "replacement bulbs"
+	icon_state = "light_box"
+	desc = "This box is shaped on the inside so that only light tubes and bulbs fit."
```

Comment:
```suggestion
	name = "replacement bulbs"
	desc = "This box is shaped on the inside so that only light tubes and bulbs fit."
	icon_state = "light_box"
```